pom.xml

```diff
@@ -160,36 +163,21 @@
                 <include>**/*</include>
               </includes>
             </fileset>
+            <fileset>
+              <directory>${project.basedir}/src/main/webapp</directory>
+              <includes>
+                <include>**/*</include>
+              </includes>
+              <excludes>
+                <exclude>css/custom-prism.css</exclude>
```

Comment:
This seems to delete the `css/custom-prism.css` file. At least it is not part of the plugin anymore.

Reply:
doesn't this exclude it from clean?